src/foam/u2/md/OverlayDropdown.js

```diff
@@ -114,78 +83,45 @@ foam.CLASS({
     function add() {
       // TODO: Replace with content @kgr
       if ( this.addToSelf_ ) {
-        this.SUPER.apply(this, arguments);
```

Comment:
Personally I don't like mixing stylistic changes with functional changes in the same PR.

But you've already written it so I guess no need to revert your changes.  Something to think about going forward.

Reply:
> Personally I don't like mixing stylistic changes with functional changes in the same PR.
> 
> But you've already written it so I guess no need to revert your changes. Something to think about going forward.

That makes sense. I usually try to do the same thing but instead of a separate PR I try to put all of the style-only changes in one commit. That way it's easy to hide all of the style changes when reviewing because you can just hide that commit when looking at the diff. Seems like different ways of doing the same thing to me, unless there's an advantage you know of to putting them in a separate PR.